packages/ui-react/src/components/Epg/Epg.module.scss

```diff
@@ -11,7 +11,12 @@ $control-z-index: $base-z-index + 2;
   width: 100%;
   height: 100%;
   padding: calc(variables.$base-spacing * 3) 0 calc(variables.$base-spacing * 3) calc(variables.$base-spacing * 3);
-  background-color: var(--epg-background-color);
+  background-color: var(--epg-background-color, var(--body-background-color));
+
+  // Planby ScrollContainer
+  &.firstFrame > div:nth-child(2) > div > div:nth-child(2) {
+    scroll-behavior: auto !important;
+  }
```

Comment:
Should we add test automation, to make sure it doesn't break when updating planBy?

Reply:
I don't think that's needed. It prevents the "scroll to now" animation when opening the page, but it no crucial feature imho